pa_config/rca.conf

```diff
@@ -97,5 +97,11 @@
       "field-data-cache-upper-bound" : 0.4,
       "shard-request-cache-upper-bound" : 0.05
     }
+  },
+
+  "usage-bucket-thresholds": {
+    "young-gen-heap-promotion-rate": [100.0, 250.0, 400.0],
+    "old-gen-heap-usage": [20.0, 40.0, 75.0],
```

Comment:
Are these %ages. For other such values (like upper/lower bound), these have been expressed as decimals e.g. `0.2 for 20%`